settings.gradle

```diff
@@ -1,2 +1,5 @@
 rootProject.name = 'gradle-plugin'
 
+if (!JavaVersion.current().java8) {
+    throw new GradleException("Build requires Java 8")
```

Comment:
The Jenkins team is preparing Java 11 support for a while now.

Reply:
Yeah, still I cannot seem to build the plugin with Java 11... And I want the build to fail fast when I am using the wrong JDK.